tests/Benchmarks/Benchmarks.csproj

```diff
@@ -44,13 +44,16 @@
     <PackageReference Include="codetitans-json" Version="1.8.3" />
     <PackageReference Include="Manatee.Json" Version="9.9.2" />
     <PackageReference Include="SpanJson" Version="1.0.7" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.2" />
+    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="4.5.1" />
```

Comment:
Why is this package required in the benchmarks?

Reply:
Removed.